plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/Xenserver625StorageProcessor.java

```diff
@@ -509,6 +510,7 @@ public Answer backupSnapshot(final CopyCommand cmd) {
             secondaryStorageUrl = cacheStore.getUrl();
             destPath = destData.getPath();
         }
+        String nfsVersion = cacheStore != null ? cacheStore.getNfsVersion() : null;
```

Comment:
@nvazquez Can you check if there are any issues when nfs version is passed as _null_ in other methods (wherever applicable).

Reply:
Sure, in this case the scripts receive the empty parameter and do not apply the `vers` parameter to the mount command:

````
2022-01-26 14:51:24    DEBUG [root] #### CLOUD enter  mountNfsSecondaryStorage ####
2022-01-26 14:51:24    DEBUG [root] mountNfsSecondaryStorage with params: {'localDir': '/var/cloud_mount/999c6598-91a4-34ef-a1cb-e9e34390aeca', 'remoteDir': '10.0.32.4:/acs/secondary/ref-trl-2462-x-M7-nicolas-vazquez/ref-trl-2462-x-M7-nicolas-vazquez-sec1', 'nfsVersion': ''}
2022-01-26 14:51:24    DEBUG [root] Successfully mounted 10.0.32.4:/acs/secondary/ref-trl-2462-x-M7-nicolas-vazquez/ref-trl-2462-x-M7-nicolas-vazquez-sec1 to /var/cloud_mount/999c6598-91a4-34ef-a1cb-e9e34390aeca
2022-01-26 14:51:24    DEBUG [root] #### CLOUD exit  mountNfsSecondaryStorage ####
2022-01-26 14:53:15    DEBUG [root] #### CLOUD enter  mountNfsSecondaryStorage ####
2022-01-26 14:53:15    DEBUG [root] mountNfsSecondaryStorage with params: {'localDir': '/var/cloud_mount/999c6598-91a4-34ef-a1cb-e9e34390aeca', 'remoteDir': '10.0.32.4:/acs/secondary/ref-trl-2462-x-M7-nicolas-vazquez/ref-trl-2462-x-M7-nicolas-vazquez-sec1', 'nfsVersion': ''}
2022-01-26 14:53:15    DEBUG [root] #### CLOUD exit  mountNfsSecondaryStorage ####
````

Reply:
thanks for confirming it @nvazquez